com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/Query.php

```diff
@@ -335,7 +335,7 @@ private function getLevelTypeOptions() {
     }
 
     private function getOptionsWithIDAsKeyFor($daoName, $field) {
```

Comment:
Since now value is used as key, you also have to change the method name to reflect this